laravel-app/resources/views/welcome.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="splash-page">
+	<div class="container">
+		<div class="row justify-content-center">
+			<div class="col-md-6">
+				<div class="text-center">
+				<img class="mr-0" src="{{ asset('assets/images/coffee-vector-min.png') }}" alt="Coffee vector">
+					<div class="logo">
+                        <span class="mi">Mi</span>
+                        <span class="cro">cro</span>
+                        <span class="blog">blog</span>
+                    </div>
```

Comment:
fix indentions

Reply:
the indentions are okay sir in the file but displays incorrectly here only
![image](https://github.com/bpocrafael/microblog-project/assets/144191038/6bbbd916-3418-4ce6-b2ba-7f1d4dfbe670)